helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -38,3 +38,10 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: PrometheusRuleFailures
+      annotations:
+        description: {{`Prometheus {{$labels.namespace}}/{{$labels.pod}} has failed to evaluate {{ printf "%.0f" $value }} rules.`}}
+        summary: Prometheus is failing rule evaluations.
+      expr: increase(prometheus_rule_evaluation_failures_total[30m]) > 0
+      labels:
+        severity: critical
```

Comment:
what does `critical` mean?

Reply:
eh sorry, that was a copy paste

Reply:
I could believe only having this in business hours would be nice too

Reply:
Adjusted labels